python/init_container.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Exit immediately if a command exits with a non-zero status, or zero if all
+# commands in the pipeline exit successfully.
+set -e
+
+# ###
+# section: Define functions
+# ###
+function initialize() {
+  echo "Initializing diagnostics..."
+
+  rm -rf /root/.dcos && mkdir /root/.dcos
+
+  cd /dcos-cli-directory
+  find . \
+    \( -name 'dcos.toml' -or -name 'attached' \) \
+    -exec cp --parents \{\} /root/.dcos \;
+  cd -
+
+  DCOS_CLUSTER_MAJOR_MINOR_VERSION=$(dcos --version | awk -F"=" '/^dcos.version=/ {split($2,verPart,".");print verPart[1]"."verPart[2]}')
```

Comment:
Not sure if you saw this comment:

> Um, what I meant was that without set -o pipefail if dcos --version fails for some reason, the failure will be ignored by the shell and DCOS_CLUSTER_MAJOR_MINOR_VERSION will end up empty,

Reply:
I saw but did not have the possibility to respond, I have checked with and without `set -o pipefail` it shows errors in both cases.

Reply:
option added